phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewLibrary.java

```diff
@@ -58,20 +57,18 @@
     private SortedList<ShortcutDTO> sortedShortcuts;
 
     private TabPane libraryTabs;
-    private Runnable onTabOpened = () -> {
-    };
+    private Runnable onTabOpened = () -> {};
 
-    private Consumer<ShortcutDTO> onShortcutSelected = shortcut -> {
-    };
-    private Consumer<ShortcutDTO> onShortcutDoubleClicked = shortcut -> {
-    };
+    private Consumer<ShortcutDTO> onShortcutSelected = shortcut -> {};
+    private Consumer<ShortcutDTO> onShortcutDoubleClicked = shortcut -> {};
 
-    public ViewLibrary(String applicationName, ThemeManager themeManager) {
+    public ViewLibrary(String applicationName, ThemeManager themeManager, ObjectMapper objectMapper) {
         super("Library", themeManager);
         this.getStyleClass().add("mainWindowScene");
+        this.objectMapper = objectMapper;
 
         availableShortcuts = new CombinedListWidget<>(ListWidgetEntry::create, (selectedItem, event) -> {
-            ShortcutDTO shortcutDTO = selectedItem;
+            final ShortcutDTO shortcutDTO = selectedItem;
```

Comment:
This whole line is not needed, because we're simply doing a rename here.
I think it would be better to use `selectedItem` instead of `shortcutDTO`.